src/pages/Home/Content/Content.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { ContentStyle } from "./ContentStyle";
+import { FoodItem, Category } from "DummyData/DataTypes";
+import { McSpicy } from "DummyData/McSpicy";
+import { OvaltineMcFlurry } from "DummyData/OvaltineMcFlurry";
+import { Categories } from "DummyData/Categories";
+import SectionHeader from "./Section/SectionHeader/SectionHeader";
+import ItemList from "./Section/ItemList/ItemList";
+
+const AllFoodItems: FoodItem[] = [
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  McSpicy,
+  OvaltineMcFlurry,
+  OvaltineMcFlurry,
+  OvaltineMcFlurry,
+];
+
+//trying to render 6 cards of identical McSpicy foodItems. Should be under the "recommended" SectionHeader.
+//McSpicy would appear both under "Recommended" and "Main Course". FoodItem can have more than one category.
```

Comment:
Can you elaborate on this? 
You can put the data on the category in the dummy data, not calculate here. For eg, you can have a dummy data for just food items. In each food object, it can contain the category. Then in the UI component you can use a filter function to find all the relevant food objects to be rendered

Reply:
Pseudo code explaining the code. Irrelevant.